helpers/installcredprovider.ps1

```diff
@@ -15,6 +16,27 @@ if ([Net.ServicePointManager]::SecurityProtocol.ToString().Split(',').Trim() -no
     [Net.ServicePointManager]::SecurityProtocol += [Net.SecurityProtocolType]::Tls12
 }
 
+$pluginLocation = "$env:USERPROFILE\.nuget\plugins"
+$tempZipLocation = "$env:TEMP\CredProviderZip"
+$localNetcoreCredProviderPath = "netcore\CredentialProvider.Microsoft"
+$localNetfxCredProviderPath = "netfx\CredentialProvider.Microsoft"
+
+# If the parameter CheckIfExists is true, check if plugin directory is already populated
```

Comment:
If you want a way to make it not install if it's already there, add a -Force switch instead. So, to always install it, the user needs to specify -Force, otherwise it'll only install if it isn't already there. If it's already there, print a message with the current version, saying to run with -Force parameter to update to the latest version.

Reply:
Good idea, thanks!